src/app/routes/utils/regex/index.js

```diff
@@ -55,6 +56,7 @@ export const mostReadPagePath = getMostReadPageRegex(allServices);
 export const mostReadDataRegexPath = getMostReadDataRegex(allServices);
 
 export const mostWatchedDataPath = getMostWatchedDataRegex(allServices);
+export const mostWatchedPagePath = getMostWatchedPageRegex(allServices);
```

Comment:
There might be a failing test here - snapshot tests are autogenerated for regex, so might need to be updated